nemo/collections/asr/metrics/rnnt_wer.py

```diff
@@ -397,6 +403,44 @@ def decode_ids_to_tokens(self, tokens: List[int]) -> List[str]:
         """
         raise NotImplementedError()
 
+    @abstractmethod
+    def decode_tokens_to_lang(self, tokens: List[int]) -> str:
+        """
+        Implemented by subclass in order to decoder a token id list into the language ID (LID) string.
+
+        Args:
+            tokens: List of int representing the token ids.
+
+        Returns:
+            A decoded LID string.
+        """
+        raise NotImplementedError()
+
+    @abstractmethod
+    def decode_ids_to_langs(self, tokens: List[int]) -> List[str]:
+        """
+        Implemented by subclass in order to decode a token id list into language ID (LID) list.
+
+        Args:
+            tokens: List of int representing the token ids.
+
+        Returns:
+            A list of decoded LIDS.
+        """
+        raise NotImplementedError()
+
+    def decode_ids_to_words_and_langs(self, tokens: List[int]) -> List[str]:
+        """
```

Comment:
This is gonna cause combinatorial explosion and is unnecessary imo. Try to remove the need for this flag by pre checks, and simply augmenting the output if necessary?

May not be possible so this is not necessary if it's not possible.